camunda-rpa/setup.py

```diff
@@ -34,5 +34,10 @@
         "Camunda.Windows",
         "Camunda.Word.Application",
     ],
-    install_requires=["rpaframework", "rpaframework-pdf", "rpaframework-windows"],
+    install_requires=[
+        "selenium==4.29.0",
```

Comment:
This still causes conflicts on the consumer side and requires manually specifying the updated transitive dependencies. Can we add the 
```
        "pillow >= 10.4.0",
        "PyYAML >= 6.0.0",
```
here as well? That seems to sort it out. 

I also wonder if we can be `>=` for the Selenium dep

Reply:
I don't see why not, let's update it 👍